decks/react/02-props.mdx

```diff
@@ -0,0 +1,79 @@
+import { FullScreenCode } from "mdx-deck/layouts";
+
+## Props
+
+- Data input from other components within the application
+- The central mechanism by which components communicate with each other
+- `props` is the name of the argument the component will receive
+- Props are used to create dynamic behaviour
+
+---
+
+export default FullScreenCode
+
+```javascript Props Example
+const Greeting = props => <h1>Hello, {props.name}!</h1>
+
+<Greeting name="Evan" />
+```
+
+---
+
+## Props
+- A prop can be anything, including a function
+- You can pass raw values or variables
+
+---
+
+export default FullScreenCode
+
+```js Evaluated Props
+const names = ['Evan', 'Renee', 'Stranger'];
+
+const ManyGreetings = props =>
+(<div>
+     { props.names.map((name, key) =>
+        <Greeting name={name} />
+     )}
+</div>)
+
+```
+
+---
+
+## Destructuring Props
```

Comment:
.... see my comment above - maybe move this a slide or two before the "anything can be a prop"